### ISSUE TITLE
When write in Universal applications Microsoft nvda does not read the input character. windows 10 version1607

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#6676]
When
write in
Universal
applications
Microsoft nvda does not read the input character.

**Comment by josephsl** [comment]
Hello,

Yes, a major issue that NV Access is indeed working on (covered by #6017). For now, I'll close this as a duplicate. Thanks.
